ext/standard/basic_functions.c

```diff
@@ -4045,6 +4050,19 @@ PHP_FUNCTION(getenv)
 }
 /* }}} */
 
+/* {{{ proto array environ(void)
+   Get all environment variables */
+PHP_FUNCTION(environ)
+{
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	array_init(return_value);
+	php_import_environment_variables(return_value);
```

Comment:
maybe you should check wether PG(http_globals)[TRACK_VARS_ENV] is available before calling import_environment.. right?